packages/react-core/src/components/EmptyState/EmptyState.tsx

```diff
@@ -10,37 +11,78 @@ export enum EmptyStateVariant {
   full = 'full'
 }
 
+export type EmptyStateStatus = 'danger' | 'warning' | 'success' | 'info' | 'custom';
+export type EmptyStateVariantType = 'xs' | 'sm' | 'lg' | 'xl' | 'full';
```

Comment:
I've gone back and forth on keeping the `EmptyStateVariant` enum, figured it could be left for a followup PR if we do want to remove it though.

I think we should decide on exporting union types like this or exporting enums more broadly before we make the call for the sake of consistency.